components/bar_buddies/BarBuddyForm.vue

```diff
@@ -39,16 +39,15 @@ nl:
         <FormInput v-model="form.pronouns" :placeholder="$t('forms.placeholder.pronouns')" />
         <FormValidation name="pronouns" :errors="validationErrors" />
       </FormElement>
-      <FormElement :label="$t('forms.label.barbuddy')" required="true">
-        <FormRadio v-model="form.barbuddy" :label="$t('forms.label.languages.no_preference')" option="no_preference" />
-        <FormRadio
+      <FormElement :label="$t('forms.label.barbuddy')" required="false">
+        <FormMultiCheckbox
```

Comment:
I think you forgot to include this component in your commit

Reply:
I did do that